src/script/src/manager.rs

```diff
@@ -112,19 +128,43 @@ impl ScriptManager {
             .context(CompilePythonSnafu { name })
     }
 
+    /// Get or create the scripts table in the catalog
+    pub fn get_scripts_table(&self, catalog: &str) -> Option<ScriptsTableRef<E>> {
+        self.tables.read().unwrap().get(catalog).cloned()
+    }
+
+    /// Insert a scripts table.
+    pub fn insert_scripts_table(&self, catalog: &str, table: TableRef) {
+        let mut tables = self.tables.write().unwrap();
+        tables.insert(
+            catalog.to_string(),
+            Arc::new(ScriptsTable::new(
+                table,
+                self.grpc_handler.load().as_ref().clone(),
+                self.query_engine.clone(),
+            )),
+        );
+    }
+
     pub async fn insert_and_compile(
```

Comment:
This function should also notify other frontend instances a new script is available. We can do this in the future

Reply:
Why? The other frontend can compile the script by themselves. They can query the script from table and compile it before executing scripts. Current implementation already supported it.